.travis.yml

```diff
@@ -28,15 +32,15 @@ jobs:
     name: "Run tests gRPC example (scala)"
 
   - script:
-    - "bin/runSbtJob shopping-cart/shopping-cart-java docker:publishLocal test"
+    - "shopping-cart/travis-scripts/build.sh shopping-cart/shopping-cart-java sbt"
```

Comment:
I don't see `shopping-cart/travis-scripts/build.sh` in the commits. Missing file?

Reply:
yep, I added it. There was also an error that I just fixed